src/python/Spline.i

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2016 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+%module spline
+%{
+#include <ignition/math/Spline.hh>
+#include <ignition/math/Helpers.hh>
+#include <ignition/math/Vector3.hh>
+#include <ignition/math/config.hh>
+%}
+
+namespace ignition
+{
+  namespace math
+  {
+    class ControlPoint;
+    class SplinePrivate;
```

Comment:
I think you don't need to forward declare `SplinePrivate` here

Reply:
You are right I don't need this declaration.